async/dune

```diff
@@ -0,0 +1,19 @@
+(* -*- tuareg -*- *)
+let coverage_rewriter =
+  match Sys.getenv "BISECT_ENABLE" with
+  | "YES" -> "bisect_ppx"
+  | _ -> ""
+  | exception Not_found -> ""
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
+(library
+  (name message_switch_async)
+  (public_name message-switch-async)
+  (libraries
+    async
+    cohttp-async
+    message-switch-core
+  )
+  %s
```

Comment:
Is this missing a `(preprocess (pps %s))`

Reply:
Although see discussion on https://github.com/xapi-project/xenopsd/pull/576/files, we might just decide to simplify this further and not use getenv at all, and trust -conditional

Reply:
Hmm, you're right, copy paste error...

Reply:
I suggest we merge this and do the simplification for profiling separately.